src/Integrations/JiraClient.ts

```diff
@@ -0,0 +1,43 @@
+import { ConnectivityStatus } from './ConnectivityStatus';
+import { Ticket } from './Ticket';
+
+export interface JiraApiOptions {
+  readonly apiKey: string;
+  readonly user: string;
+  readonly baseUrl: string;
+}
+
+export const JiraApiOptions: unique symbol = Symbol('JiraApiOptions');
```

Comment:
It doesn't relate to the interface. I guess it must be placed in the file of implementation